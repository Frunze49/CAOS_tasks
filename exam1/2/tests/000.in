libm.so.6 float sinf 0.5
libc.so.6 int putchar 37
