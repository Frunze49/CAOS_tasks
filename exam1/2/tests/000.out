0.479426
%37
